README.md

```diff
@@ -0,0 +1 @@
+This is a youtube clone.
```

Comment:
KIndly make sure to clone the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) in your repo in order to make the repo looks more professional and try to customize it according to your project.

**Make sure your README looks more professional by defining the headers of each section. Kindly follow the provided template to achieve it**